URBNSwiftAlert/Classes/AlertView.swift

```diff
@@ -196,6 +199,10 @@ extension AlertView: AlertButtonContainer {
             }
         }
     }
+    
+    public func addReturnKeyboardHandler(_ handler: @escaping EmptyHandler) {
```

Comment:
Wondering why its preferable to set the `returnButtonHandler` through a function rather than making it `public` and being able to set it directly?

Reply:
Fair point. Let me see how that works out